ngraph_bridge/ngraph_builder.cc

```diff
@@ -2015,6 +2015,19 @@ static Status TranslatePadOp(const Node* op,
   return Status::OK();
 }
 
+static Status TranslateRangeOp(const Node* op,
+                               const std::vector<const Tensor*>&,
+                               Builder::OpMap& ng_op_map) {
+  ng::Output<ng::Node> ng_start, ng_stop, ng_step;
+  TF_RETURN_IF_ERROR(GetInputNodes(ng_op_map, op, ng_start, ng_stop, ng_step));
```

Comment:
start, stop and step should be static inputs.

Reply:
Why? Those values are not needed to be known before hand

Reply:
The inputs probably need to be static.